code/modules/mob/living/simple_animal/hostile/netherworld.dm

```diff
@@ -101,11 +126,44 @@
 		if(M)
 			playsound(src, 'sound/magic/demon_consume.ogg', 50, TRUE)
 			M.adjustBruteLoss(60)
-			new /obj/effect/gibspawner/generic(get_turf(M), M)
 			if(M.stat == DEAD)
 				var/mob/living/simple_animal/hostile/netherworld/blankbody/blank
 				blank = new(loc)
-				blank.name = "[M]"
-				blank.desc = "It's [M], but [M.p_their()] flesh has an ashy texture, and [M.p_their()] face is featureless save an eerie smile."
-				visible_message("<span class='warning'>[M] reemerges from the link!</span>")
-				qdel(M)
\ No newline at end of file
+				blank.name = "[M.real_name]"
+				blank.desc = "It's [M.real_name], but [M.p_their()] flesh has an ashy texture, and [M.p_their()] face is featureless save an eerie smile."
+				blank.maxHealth = 150
+				blank.health = 150
+				blank.melee_damage_upper = 15
+				visible_message("<span class='warning'>[M.real_name] reemerges from the link!</span>")
+				if((blankckey) && (blankckey == M.ckey))
+					blank.ckey = blankckey
+					blankckey = null
+					to_chat(blank, "<span class='biggerdanger'>Hatred courses through you... you can't give up your spirit... not yet. Help your newfound compatriots complete their objectives!</span>")
+					SSticker.mode.add_cultist(blank.mind, 1)
+					blank.mind.special_role = "Cultist"
+				qdel(M)
+				blankckey = null
```

Comment:
So.

Why'd you decide that putting an antag giver (Which is prone to abuse and is why wish granter was made a very hard to enter ruin in the first place) a good design idea?

This encourages people to actively seek out netherworld portal and legitimately self-antag